src/ActiveLogin.Authentication.BankId.AspNetCore.Azure/KeyVault/AzureKeyVaultCertificateClient.cs

```diff
@@ -77,6 +93,24 @@ public async Task<X509Certificate2> GetX509Certificate2Async(string keyVaultSecr
             return GetX509Certificate2(certificateBytes);
         }
 
+        private async Task<X509Certificate2> CertificateFromKeyVaultCertificate(string keyVaultSecretIdentifier)
```

Comment:
Probably a copy paste thing. `keyVaultSecretIdentifier` should be `keyVaultCertificateIdentifier`

Reply:
Sorry for all these sloppy PR's. It seems my confidence is much better than my proof reading skills.

Reply:
Haha, na, no problem. That's why we do Code Reviews :) I'm the same myself!